server/views/admin-add-category.hbs

```diff
@@ -43,7 +47,7 @@
       <h3 class="text-primary">Icon:</h3>
     </label>
     <div class="col-lg-6 col-md-8 col-sm-9 col-xs-9 ">
-      <input class="form-control" type="text" name="icon" id="example-text-input">
+      <input id="iconName" class="form-control" type="text" name="icon" id="example-text-input">
```

Comment:
you know you can prevent an empty input from being submitted by just adding attribute required to the input? `<input required="true" ...>`

Reply:
I did it in mongo schema

Reply:
if you have it in mongo schema - it means that the data has to go from client to server unnecessarily. You can just prevent it from the beginning.

Reply:
please close this PR we do not need

Reply:
it's same PR number 113 
I did it in another way